[16.04] AssertionError: Multiple .dist-info directories

Installing mock v4 fails on trusty:
```
 Exception:
 Traceback (most recent call last):
   File "/usr/lib/python2.7/dist-packages/pip/basecommand.py", line 209, in main
     status = self.run(options, args)
   File "/usr/lib/python2.7/dist-packages/pip/commands/install.py", line 335, in run
     prefix=options.prefix_path,
   File "/usr/lib/python2.7/dist-packages/pip/req/req_set.py", line 732, in install
     **kwargs
   File "/usr/lib/python2.7/dist-packages/pip/req/req_install.py", line 837, in install
     self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
   File "/usr/lib/python2.7/dist-packages/pip/req/req_install.py", line 1039, in move_wheel_files
     isolated=self.isolated,
   File "/usr/lib/python2.7/dist-packages/pip/wheel.py", line 344, in move_wheel_files
     clobber(source, lib_dir, True)
   File "/usr/lib/python2.7/dist-packages/pip/wheel.py", line 304, in clobber
     assert not info_dir, 'Multiple .dist-info directories'
 AssertionError: Multiple .dist-info directories
 You are using pip version 8.1.1, however version 20.0.2 is available.
 You should consider upgrading via the 'pip install --upgrade pip' command.
```

--- a/setup.py
+++ b/setup.py
@@ -29,7 +29,7 @@ setup(
         'catkin_tools',
         'colorlog',
         'future',
-        'mock',
+        "mock < 4; python_version < '3'",
         'six>=1.7',  # https://github.com/testing-cabal/mock/issues/257
         'rosdep',
         'rosdistro >= 0.7.3',

--- a/src/ros_get/utils.py
+++ b/src/ros_get/utils.py
@@ -7,11 +7,15 @@ from rosdep2.main import _rosdep_main
 from rosdistro import get_index, get_index_url, repository, get_distribution, DEFAULT_INDEX_URL
 from rosdistro.source_repository_specification import SourceRepositorySpecification
 
-from mock import patch
 from rosdep2 import RosdepLookup, create_default_installer_context, get_default_installer
 from rosdep2.rospkg_loader import DEFAULT_VIEW_KEY
 from rosinstall_generator.generator import generate_rosinstall_for_repos
 from vcstools import get_vcs_client
+
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
 
 logger = logging.getLogger(__name__)
 

